guides/common/attributes-satellite.adoc

```diff
@@ -110,15 +110,15 @@
 :RepoSatelliteVersion: {ProductVersion}
 
 // RHEL 9 Satellite repos
-:RepoRHEL9ServerSatelliteToolsProductVersion: satellite-client-6-for-rhel-9-<arch>-rpms
```

Comment:
I think the idea of `<arch>` is that we support multiple arches (like `x86_64`), and this indicates that. I'm not sure which arches exactly, but perhaps we should list them somewhere to make it easier to consume.

Reply:
Also, if you decide to go through with this, you need to change line 27 in [proc_registering-clients-using-the-bootstrap-script.adoc](https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_registering-clients-using-the-bootstrap-script.adoc):
```
<1> Replace `<arch>` with the client architecture, for example `x86`.
```